docs/index.rst

```diff
@@ -34,19 +36,26 @@ ImageCraft
       **Discussion and clarification** of key topics
 
 Project and community
-=====================
+---------------------
+
+Imagecraft is a member of the Canonical family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
+
+Get started:
+  * Read our `Code of Conduct`_
 
-Imagecraft is a member of the Canonical family. It's an open source project
-that warmly welcomes community projects, contributions, suggestions, fixes
-and constructive feedback.
+Discuss:
+  * Discourse: `Ubuntu Foundations`_
 
-* `Ubuntu Code of Conduct <https://ubuntu.com/community/code-of-conduct>`_.
-* `Canonical contributor licenses agreement
-  <https://ubuntu.com/legal/contributors>`_.
+..   * IRC: `Libera.Chat`_, the *#ubuntu-server* channel
```

Comment:
Is that commented on purpose?

Reply:
Yes, but I wanted to ask about a suitable IRC channel.